models/organization/team.go

```diff
@@ -123,35 +123,20 @@ func (opts *SearchTeamOptions) toCond() builder.Cond {
 func SearchTeam(opts *SearchTeamOptions) ([]*Team, int64, error) {
 	sess := db.GetEngine(db.DefaultContext)
 
-	opts.SetDefaultValues()
 	cond := opts.toCond()
 
 	if opts.UserID > 0 {
 		sess = sess.Join("INNER", "team_user", "team_user.team_id = team.id")
 	}
 
-	count, err := sess.
-		Where(cond).
-		Count(new(Team))
-	if err != nil {
-		return nil, 0, err
-	}
-
-	if opts.UserID > 0 {
-		sess = sess.Join("INNER", "team_user", "team_user.team_id = team.id")
-	}
-
-	if opts.PageSize == -1 {
-		opts.PageSize = int(count)
-	} else {
-		sess = sess.Limit(opts.PageSize, (opts.Page-1)*opts.PageSize)
+	if opts.PageSize != -1 {
```

Comment:
Isn't that always true and can therefore be removed?

Reply:
as models/** func are our internal datastore api ... I would let this in as it is common behavior ...

we might refactor it, so SetDefaultValues does respect this by default but, that needs regression testing and wont be scope of this pull

Reply:
> Isn't that always true and can therefore be removed?

I didn't go throu all func's calling this func - so I cant say for sure atm, but that does not matter

Reply:
I have seen 4 calls which all provide the `ListOptions` and even if it's not set the default value is 0.

Reply:
well I dont mind to enforce it for now until one needs a non paginated one